### PR TITLE
[#155861133] Bug fix load test

### PR DESCRIPTION
**Pull request by mpope9**
Lots of parts of the load test were broken/didn't reflect reality.  To list a few it didn't reflect the background polling of offline messages, there were no connections to random muc rooms, all messages were sent to room 81 regardless of being in room 81, all clients had the exact same resource, typos, id's were being reused, and the built in tsung things are just not powerful enough for our purposes.

Now its functioning and tests load!

@kkaminski

### CODE REVIEW

File: tsung.xml [hunk 1]
```diff
@@ -61,56 +64,79 @@
...
+         <thinktime value="3" random="false" />
 
-         <transaction name="presence_broadcast_all">
-            <request>
-               <jabber type="presence:broadcast" show="chat" status="Available to chat" ack="no_ack"/>
-            </request>
-         </transaction>
-         <!-- keep alive -->
-         <thinktime value="120" random="false" />
+         <for from="0" to="600" incr="1" var="counter">
```
Comment (leejonaNEU) [inline tsung.xml]: nit you use `var="i"` for most loops, consistency is good

File: tsung.xml [hunk 2]
```diff
@@ -61,56 +64,79 @@
...
-            <thinktime value="30" random="true" />
+            <thinktime value="10" random="true" />
+         </for>
+         <for from="0" to="7" incr="1" var="counter">
+            <transaction name="get_offline_messages">
+               <request subst="true">
+                   <jabber type="raw" ack="no_ack" data="&lt;iq   xmlns=&quot;jabber:client&quot; type=&quot;get&quot; id=&quot;%%ts_user_server:get_unique_id%%&quot;&gt;   &lt;query     xmlns=&quot;http://jabber.org/protocol/disco#info&quot; node=&quot;http://jabber.org/protocol/offline&quot;/&gt;   &lt;/iq&gt;" />
+               </request>
+               <request subst="true">
+                  <jabber type="raw" ack="no_ack" data="&lt;iq   xmlns=&quot;jabber:client&quot; type=&quot;get&quot; id=&quot;%%ts_user_server:get_unique_id%%&quot;&gt;   &lt;query     xmlns=&quot;http://jabber.org/protocol/disco#items&quot; node=&quot;http://jabber.org/protocol/offline&quot;/&gt;   &lt;/iq&gt;" />
```
[the remark this answers is not in the document]
Comment (mpope9) [reply tsung.xml]: fixed

File: tsung.xml (hunk 1 above)
Comment (mpope9) [reply tsung.xml]: fixed

**Comment by kkaminski** [comment]
@mpope9 I added https://www.pivotaltracker.com/story/show/156691863 and tied it to the chat slowness epic so we don't lose track of this.

**Pull request merged**